tests/dummy/app/services/i18n.js

```diff
@@ -8,7 +8,7 @@ export default Service.extend({
 	rootURL: ENV.rootURL,
 
 	initialize(language) {
-		return fetch(`${this.rootURL}/assets/i18n/${language}/design-system.json`)
+		return fetch(`${this.rootURL}assets/i18n/${language}/design-system.json`)
```

Comment:
Reference page didn't want to run because there were to slashes at the beginning of this path.